doc/develop/getting_started/index.rst

```diff
@@ -476,6 +476,21 @@ Zephyr applications.
 It also contains additional host tools, such as custom QEMU and OpenOCD builds
 that are used to emulate, flash and debug Zephyr applications.
 
+.. note::
+   Before installing, check whether the :envvar:`ZEPHYR_TOOLCHAIN_VARIANT` or
+   :envvar:`ZEPHYR_SDK_INSTALL_DIR` environment variables are already set.
+
+   * If the variables are not set, the latest compatible version of the Zephyr SDK will be selected by default.
+     Proceed to next step without making any changes.
+
+   * If :envvar:`ZEPHYR_TOOLCHAIN_VARIANT` is set, the corresponding toolchain will be selected at build time
+     instead of the Zephyr SDK. Either unset the :envvar:`ZEPHYR_TOOLCHAIN_VARIANT` environment variable or change its
+     value to ``zephyr`` to make sure the Zephyr SDK is selected.
+
+   * If the :envvar:`ZEPHYR_SDK_INSTALL_DIR` environment variable is set, it will override the lookup location
+     for the selected toolchain. If you install the Zephyr SDK to one of the locations recommended below,
+     you can unset this variable. Otherwise, set it to your chosen install location.
```

Comment:
I don't think this belongs here.

New users starting out with Zephyr or installing on a clean machine doesn't need to carry out those steps. They are imho just noise in those cases.

We should try to keep the guide as simple as possible without unnecessary steps for newcomers.

Instead this belongs in an *Updating Zephyr SDK* section or a trouble shooting section as this may only happen to users updating an existing system (which is beyond getting started).

If you add a *Update Zephyr SDK* section to the [Zephyr SDK](https://docs.zephyrproject.org/latest/develop/toolchains/zephyr_sdk.html) page then you can simple change the note to:
```suggestion
.. note::
   If you're updating the Zephyr SDK from an older release, please read :ref:`update_sdk`
``` 

Another option is to change the note to pure text and move it downwards to a *Troubleshooting* subsection. That subsection can then include the [asking-for-help](https://docs.zephyrproject.org/latest/develop/getting_started/index.html#asking-for-help) subsection.

This will be more aligned with what we have for [devicetree](https://docs.zephyrproject.org/latest/build/dts/index.html) and [west](https://docs.zephyrproject.org/latest/develop/west/index.html) where there exists dedicated [Troubleshooting](https://docs.zephyrproject.org/latest/develop/west/troubleshooting.html) sections for users experiencing problems.

Reply:
I went for adding a Troubleshooting section.